boot/freeldr/freeldr/lib/fs/fs.c

```diff
@@ -1,7 +1,7 @@
 /*
  *  FreeLoader
  *  Copyright (C) 1998-2003  Brian Palmer  <brianp@sginet.com>
- *  Copyright (C) 2008-2009  Herv� Poussineau  <hpoussin@reactos.org>
+ *  Copyright (C) 2008-2009  Herv� Poussineau  <hpoussin@reactos.org>
```

Comment:
Can you confirm that now the file is in UTF-8 (without BOM)? (If so then this is OK)